wallet-test/src/test/scala/org/bitcoins/wallet/util/BitcoinSWalletTest.scala

```diff
@@ -57,21 +58,44 @@ trait BitcoinSWalletTest
       .map(_ => ())
   }
 
-  def createNewWallet(): Future[UnlockedWalletApi] = {
-
-    for {
-      _ <- config.initialize()
-      wallet <- Wallet.initialize().map {
-        case InitializeWalletSuccess(wallet) => wallet
-        case err: InitializeWalletError =>
-          logger.error(s"Could not initialize wallet: $err")
-          fail(err)
-      }
-    } yield wallet
-  }
+  private val createNewWallet: Option[Config] => () => Future[
+    UnlockedWalletApi] =
+    extraConfig =>
+      () => {
+        val defaultConf = implicitly[WalletAppConfig]
```

Comment:
I don't really understand why this is here? Isn't this reading the default config in this trait? In general it seems like we wanted to use implicits, but now are passing them all explicitly which kind of defeats the point IMO.